app/src/main/res/layout/drawer_header.xml

```diff
@@ -3,14 +3,15 @@
     xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     android:id="@+id/navigation_drawer_header"
-    android:layout_width="wrap_content"
+    android:layout_width="match_parent"
     android:layout_height="@dimen/nav_drawer_header_height">
 
-    <ImageView
+    <LinearLayout
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:scaleType="centerCrop"
-        android:src="@drawable/navigation_drawer_header_background" />
+        android:gravity="center"
+        android:background="@color/material_black"/>
```

Comment:
I've got the idea of using the primaryDark color for this from here:
https://github.com/chrisbanes/cheesesquare/blob/master/app/src/main/res/layout/nav_header.xml#L21

If you think, using the attribute `?attr/colorPrimaryDark` may cause problems, the resource `@color/primary_dark` could be used instead.

Just using a new color seems a bit arbitrary to me.
(primaryDark is `#313131` but the new one is `#212121`)